### ISSUE TITLE
Vulnerability to XSS

### ISSUE THREAD

**Issue by gjc13** [openthread/borderrouter#157]
```
GET /no_such_file.xxx?"><script>alert(73541);</script> HTTP/1.1
Host: 127.0.0.1
Connection: Keep-Alive
```
Causes the web page to alert

**Comment by gjc13** [comment]
Temporarily limiting access to localhost in this commit:https://github.com/gjc13/borderrouter/commit/cc1a253773007a42fdae0ec556c109eb4b5b79b0
The vulnerability, however, comes from the http framework we are using now. To totally fix this issue, maybe we should wait for the api definition of openthread and implement the web server with modern frameworks.

**Comment by bukepo** [comment]
Is it reasonable to provide a match all resource responding with escaped information as a quick fix for this issue?

**Comment by jwhui** [comment]
Resolved by #162.
